tensorflow/python/keras/metrics.py

```diff
@@ -682,6 +682,44 @@ def from_config(cls, config):
     return super(MeanMetricWrapper, cls).from_config(config)
 
 
+class SparseCategoricalMetricWrapper(MeanMetricWrapper):
```

Comment:
I think having a dedicated class just for having different docstring is bit too overkill, and also increase maintaince overhead. Can we just put the docstring as const above?

Reply:
Sure can do. Updated the code as instructed.